src/main/python/model/model.py

```diff
@@ -27,38 +27,13 @@ def __init__(self, mainTableModel: QStandardItemModel):
 
         self._mainTableModel = mainTableModel
 
-    def load_file(self, fileName):
-        self._mainTableModel.clear()
-        # TODO : sync View with RecordTable
-        with open(fileName, "r") as csvfile:
-            reader = csv.reader(csvfile, delimiter=";", lineterminator="\n")
-            headers = reader.__next__()
-            for row in reader:
-                items = [QStandardItem(field) for field in row]
-                self._mainTableModel.appendRow(items)
-            for i, header in enumerate(headers):
-                self._mainTableModel.setHeaderData(i, Qt.Orientation.Horizontal, header)
-
-            if set(FlightDatasManager.STATE_FLIGHT_DATASET.get_keys()).issubset(
-                headers
-            ):
-                FlightDatasManager.set_dataset_as_state()
-            else:
-                FlightDatasManager.clean_dataset()
-            for h in headers:
-                # TODO : add unit in loaded files or manage them later
-                FlightDatasManager.add_data(h, None)
-
-            if self.status != ModelStatus.OFFLINE:
-                self._player.reset()
-
     def save_file(self, fileName):
-        save_datas(fileName, self._mainTableModel)
+        save_sr(fileName, self._mainTableModel)
 
-    def import_file(self, fileName):
+    def open_file(self, fileName):
         self._mainTableModel.clear()
+        open_sr(fileName, self._mainTableModel)
         FlightDatasManager.set_dataset_as_state()
```

Comment:
`FlightDatasManager` is not properly updated (ex: if the .sr file doesn't contain all state datas)
 Maybe the previous code shall be reused